src/IBM.Watson.CompareComply.v1/Model/KeyValuePair.cs

```diff
@@ -31,10 +30,10 @@ public class KeyValuePair
         [JsonProperty("key", NullValueHandling = NullValueHandling.Ignore)]
         public Key Key { get; set; }
         /// <summary>
-        /// A list of values in a key-value pair.
+        /// A value in a key-value pair.
         /// </summary>
         [JsonProperty("value", NullValueHandling = NullValueHandling.Ignore)]
-        public List<Value> Value { get; set; }
```

Comment:
Looks like this got reverted

Reply:
Yeah, I generated using the last commit we generated from to keep the changes minimal. I forgot this was a hand-edit!